blog/rfcs.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Introducing electron/rfcs
+date: 2024-02-13T12:00:00.000Z
+authors:
+  - name: erickzhao
+    url: 'https://github.com/erickzhao'
+    image_url: 'https://github.com/erickzhao.png?size=96'
+slug: rfcs
+---
+
+Electron’s [API Working Group](https://github.com/electron/governance/tree/main/wg-api) is
+adopting an open **Requests for Comments (RFC)** process to help shepherd larger changes
+to Electron core.
+
+## Why RFCs?
+
+In short, we want to smooth out the process of landing significant changes to Electron core.
+
+Currently, new code changes are mostly discussed through issues and pull requests on GitHub.
+For most changes to Electron, this is a good system. Many bug fixes, documentation changes,
+and even new features are straightforward enough to review and merge asynchronously through
+standard GitHub flows.
+
+For changes that are more significant—for instance, large API surfaces or breaking changes
+that would affect the majority of Electron apps—it makes sense for review to happen at the
+ideation stage before most of the code is written.
+
+This process is designed to be open to the public, which will also make it easier for the
+open source community at large to give feedback on potential changes before they land in
+Electron.
+
+## How does it work?
+
+```mermaid
+flowchart TD
+  P[Proposed] --> PC[Pending Comment];
+  PC --> A[Active];
+  A --> C[Completed];
+  PC --> P;
+  P --> D[Declined];
+  A --> D;
+  style D fill:red;
+  style A fill:orange;
+  style C fill:lightgreen;
+  style PC fill:yellow;
```

Comment:
```suggestion
  style P color:black;
  style P fill:white;
  style D color:black;
  style D fill:red;
  style A color:black;
  style A fill:orange;
  style C color:black;
  style C fill:lightgreen;
  style PC color:black;
  style PC fill:yellow;
```

The current styling is difficult to read in dark mode due to the way the `@docusaurus/theme-mermaid` package selects the text color such that it doesn't have much contrast with the color fills.

Explicitly set the text color (and set a white fill for `P`) so that it is legible in both light and dark mode.

Before:

<img width="282" alt="Screenshot 2024-02-12 at 11 19 13 PM" src="https://github.com/electron/website/assets/5820654/4bcc1440-d876-4190-bd5d-289ef639abe9">
<img width="270" alt="Screenshot 2024-02-12 at 11 19 26 PM" src="https://github.com/electron/website/assets/5820654/ce4cd70c-12c7-4bcf-995d-6b73afb0baae">

After:

<img width="285" alt="Screenshot 2024-02-12 at 11 19 59 PM" src="https://github.com/electron/website/assets/5820654/aaaa8a4c-4e65-442e-be7b-60fd1520e7d9">
<img width="269" alt="Screenshot 2024-02-12 at 11 20 10 PM" src="https://github.com/electron/website/assets/5820654/122d1066-c13e-4ed3-9e77-4a3d49c2a3a8">